addon/mixins/in-viewport.js

```diff
@@ -177,22 +219,21 @@ export default Mixin.create({
     debounce(this, () => this[methodName](...args), get(this, 'viewportRefreshRate'));
   },
 
-  _bindScrollDirectionListener(context = null, sensitivity = 1) {
-    assert('You must pass a valid context to _bindScrollDirectionListener', context);
+  _bindScrollDirectionListener(sensitivity = 1) {
     assert('sensitivity cannot be 0', sensitivity);
 
-    const $contextEl = $(context);
+    const $contextEl = get(this, 'scrollableArea') ? $(get(this, 'scrollableArea')) : $(window);
 
     $contextEl.on(`scroll.directional#${get(this, 'elementId')}`, () => {
       this._debouncedEventHandler('_triggerDidScrollDirection', $contextEl, sensitivity);
     });
   },
 
-  _unbindScrollDirectionListener(context = null) {
-    assert('You must pass a valid context to _bindScrollDirectionListener', context);
-
+  _unbindScrollDirectionListener() {
     const elementId = get(this, 'elementId');
 
+    const context = get(this, 'scrollableArea') ? get(this, 'scrollableArea') : window;
```

Comment:
Could probably make a variable on this mixin instead of this ternary.

Reply:
`const context = get(this, 'scrollableArea') || window` ?